system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -43,6 +44,74 @@
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+         (is (= 201 (:status response)))
+         (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))
+     (testing "ingest on PROV1 with subscription change for subscriber-id below subscription limit"
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-6"
+                                                                   :Name "sub1"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+        (is (= 200 (:status response)))
+        (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))
+     (testing "ingest on PROV1 with subscription request for subscriber-id below subscription limit"
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-7"
+                                                                   :Name "sub2"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+        (is (= 201 (:status response)))
+        (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))))
+
+(deftest subscription-count-exceeds-limit-test
+  (side/eval-form `(jobsub/set-subscriptions-limit! 1))
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-1"
+                 :EntryTitle "entry-title-exceed-limit-1"})
+               {:token "mock-echo-system-token"})
+        user1-token (echo-util/login (system/context) "user1")]
+     (testing "ingest on PROV1 with subscription request for subscriber-id below subscription limit" ;zzz
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-7"
+                                                                   :Name "sub1"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+         (is (= 201 (:status response)))
+         (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))
+     (testing "ingest on PROV1 with subscription request for subscriber-id already at subscription limit"
```

Comment:
how about renaming to "fails with correct response when ingesting subscription for user beyond their subscription limit"

Reply:
Definitely sounds much better that way, reworded.